examples/fairmotion_interface.py

```diff
@@ -470,6 +463,32 @@ def cycle_model_previews(self) -> None:
         self.toggle_key_frames()
         self.build_trajectory_vis()
 
+    def belongs_to(self, obj_id: int) -> bool:
+        """
+        Accepts an object id and returns True if the obj_id belongs to an object
+        owned by this Fairmotion character.
+        """
+        # checking our model links
+        if self.model and obj_id in self.model.link_object_ids:
+            return True
+
+        # checking our model
+        if self.model and obj_id is self.model.object_id:
+            return True
+
+        # checking all key frame models
+        if any(
+            obj_id in ko_ids
+            for ko_ids in [i.link_object_ids.keys() for i in self.key_frame_models]
```

Comment:
Nested comprehension should probably be a set comprehension
```suggestion
            for ko_ids in {i.link_object_ids.keys() for i in self.key_frame_models}
```

Reply:
Ah, I misinterepreted what "in" means in this context. No reason to store a temporarly list like that though, better just to chain the generators.

```suggestion
            for ko_ids in (i.link_object_ids.keys() for i in self.key_frame_models)
```